src/app/(app)/map.web.tsx

```
@@ -20,6 +20,7 @@ import { type MapMakerInfoData } from '@/models/v4/mapping/getMapDataAndMarkersD
 import { type GetMapLayersData } from '@/models/v4/mapping/getMapLayersResultData';
 import { type PoiLayerData } from '@/models/v4/mapping/poiLayerData';
 import { useLocationStore } from '@/stores/app/location-store';
+import { getDepartmentMapCenter } from '@/lib/map-center';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)